.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.6"
```

Comment:
Should be the same as whats in the pipfile/dockerfile. Could even go to 3.8 but I don't think any of our apps are on that yet so put it to 3.7